model/generate.go

```diff
@@ -270,10 +270,16 @@ func (g *GeneratedProject) saveNewBuildsAndTasks(ctx context.Context, v *Version
 		return errors.Errorf("project '%s' not found", p.Identifier)
 	}
 
+	allTasksToBeCreatedIncludingDeps, err := NewPatchTaskIdTable(p, v, *newTVPairs, projectRef.Identifier)
```

Comment:
The variable name is good but can we also add a comment saying that this represents all tasks, both for existing builds and new builds